### DISCUSSION TITLE
How to change the server port?

### DISCUSSION THREAD

**Discussion by Mohit-3196** [ant-media/Ant-Media-Server#5533] [FAQs] [answered]
I have installed Ant Media Server and have enabled SSL and currently, I can access it on port 5443.
Is there a way to change the default port value to some other port instead of port 5443? If yes, then how?

Thank you.

**Comment by yashtandon113** [comment, marked as answer]
@Mohit-3196 
Thanks for your question. 

1. In order to change the HTTPS port from 5443 to 443 and HTTP port from 5080 to port 80, you can use the port forwarding. Please check out [**here**](https://antmedia.io/docs/guides/installing-on-linux/installing-ams-on-linux/#forward-default-http-80-https-443-ports-to-5080-and-5443).

2. In order to change the HTTPS port other then 443, go to **/usr/local/antmedia/conf** folder and edit the **red5.properties** file to change the https port as below.

  `https.port=5443`                        // By default it is 5443 but you can change as per your requirement.

After making the change, restart the server.

3. To change the HTTP port from 5080 to any other port excluding port 80, check the solution on below link.
https://antmedia.io/docs/faq/#how-can-i-change-the-default-http-port-5080

I hope it helps.
